jbmc/regression/jbmc/assume-input-limits/arg_assume_no_zero.desc

```diff
@@ -0,0 +1,23 @@
+CORE
+My.class
+--function My.numericalArg --java-assume-input-limits [1:3]
```

Comment:
:question: Not clear on what this test checks? There is no assert that can be reached by only using 0?

Reply:
I changed the tests a bit - all desc files check for positive limits only `[1:3]` and there is a specific test fro checking that negative limit works too `arg_assume_neg`